charts/ocis/values.yaml

```diff
@@ -21,170 +47,397 @@ image:
   # You can use the endpoint on your machine by forwarding the port, eg: `kubectl port-forward -n ocis pod/proxy-8587dc9d64-fs24l 9205:9205`
   # and then accessing the port on https://localhost:9147/debug/pprof or using the pprof command line tool: `go tool pprof -web http://localhost:9205/debug/pprof/symbol\?seconds\=10`
   profiling: false
 
-# -- Deployment strategy.
+# Deployment strategy.
 deploymentStrategy:
+  # @schema
+  # enum: [RollingUpdate, Recreate]
+  # required: true
+  # @schema
+  # -- deployment strategy type to use. Can be "RollingUpdate" or "Recreate"
   type: RollingUpdate
+  rollingUpdate:
+    # @schema
+    # type: string
+    # required: true
+    # @schema
+    # -- The maximum number of pods that can be scheduled above the desired number of pods.
+    # Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
+    maxSurge: 25%
+    # @schema
+    # type: string
+    # required: true
+    # @schema
+    # -- The maximum number of pods that can be unavailable during the update.
+    # Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
+    maxUnavailable: 25%
 
+# @schema
+# type: string
+# required: true
+# @schema
 # -- Domain where oCIS is reachable for the outside world
-externalDomain: ""
+externalDomain:
 
 # Insecure options.
 # These are useful for some limited environments like CI or on a test cluster.
 insecure:
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   # -- Disables SSL certificate checking for connections to the openID connect identity provider.
   # Not recommended for production installations.
   oidcIdpInsecure: false
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   # -- Disables SSL certificate checking for connections to the oCIS http apis.
   # Not recommended for production installations.
   ocisHttpApiInsecure: false
+  # @schema
+  # type: boolean
+  # required: true
+  # @schema
   # -- Disables SSL certificate checking for connections to all OCM instances
   # Not recommended for production installations.
   ocmInsecure: false
 
 cache:
+  # @schema
+  # type: string
+  # required: true
+  # @schema
   # -- Type of the cache to use.
   # Can be set to "nats-js-kv" or "redis-sentinel". There are also the non-recommended options "memory" and "noop".
   # The address of NATS / Redis Sentinel node(s) needs to be set to `cache.nodes`.
   type: "nats-js-kv"
+  # @schema
+  # type: array
+  # items:
+  #   type: string
+  #   required: true
+  # @schema
   # -- Nodes of the cache to use.
   nodes:
     - "{{ .appNameNats }}:9233"
 
 store:
+  # @schema
+  # type: string
+  # required: true
+  # @schema
   # -- Configure the store type.
   # Can be set to "nats-js-kv" or "redis-sentinel". There is also the non-recommended option "memory".
   # The address of NATS / Redis Sentinel node(s) needs to be set to `cache.nodes`.
   type: nats-js-kv
```

Comment:
To be consistent we can put this in quotation marks similar to line 328

Reply:
I removed quotes from all places where they don't have a function. This was already the case for most strings anyways.

changed in https://github.com/owncloud/ocis-charts/pull/811/commits/26845bc16c5247e3c0cb9f1ed97328a25f6d5b8c